app/src/main/java/ch/epfl/sweng/zuluzulu/Fragments/SuperFragment.java

```diff
@@ -0,0 +1,28 @@
+package ch.epfl.sweng.zuluzulu.Fragments;
+
+import android.content.Context;
+import android.support.v4.app.Fragment;
+
+import ch.epfl.sweng.zuluzulu.OnFragmentInteractionListener;
+
+public class SuperFragment extends Fragment {
+    protected OnFragmentInteractionListener mListener;
+
```

Comment:
@Globoxx @njomeau 
I created a super fragment class. It implements the listener. So we don't have to do it on each fragment and we can remove methods onAttach and onDetach on each fragment. So -10 lines per fragment.

Is it ok for you ?